cmd/stack_addtorepo.go

```diff
@@ -0,0 +1,274 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"gopkg.in/yaml.v2"
+)
+
+func newStackAddToRepoCmd(rootConfig *RootCommandConfig) *cobra.Command {
+	var releaseURL string
+	var useLocalCache bool
+	var repoName string
+
+	var stackAddToRepoCmd = &cobra.Command{
+		Use:   "add-to-repo",
+		Short: "Add stack information into a production Appsody repository",
+		Long: `This command is a tool for stack publishers to add stack information into a production index file. The add-to-repo command performs the following:
```

Comment:
Steve, please can you apply the standard help text format to this command, including examples (see #463)

Reply:
@helenmasters I have updated the help text and examples after discussing with Neeraj. Please can you re-review this.